### PR TITLE
notifications UI

### PR DESCRIPTION
**Pull request by DanielW1987**

### CODE REVIEW

File: webapp/src/main/resources/static/ts/src/clients/notification-settings-rest-client.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    private readonly NOTIFICATION_SETTINGS_URL = "/rest/v1/notification-config"
+
+    public async fetchNotificationSettings(): Promise<NotificationSettings> {
+        return await axios.get(
+          this.NOTIFICATION_SETTINGS_URL, axiosConfig
+        ).then((response: AxiosResponse<NotificationSettings>) => {
+            return response.data;
+        }).catch((error: AxiosError) => {
+            console.error(error);
```
Comment (hedgehog1833) [inline webapp/src/main/resources/static/ts/src/clients/notification-settings-rest-client.ts:15]: You want to leave the logging here and in the other method?

File: webapp/src/main/resources/static/ts/src/service/account-details-render-service.ts [hunk 2]
```diff
@@ -6,6 +6,8 @@ export class AccountDetailsRenderService {
     private readonly toastService: ToastService;
     private readonly accountDetailsRestClient: AccountDetailsRestClient;
 
+
+
```
Comment (hedgehog1833) [inline webapp/src/main/resources/static/ts/src/service/account-details-render-service.ts:10]:
```suggestion
```

File: webapp/src/main/resources/static/ts/src/service/notification-settings-render-service.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        this.notificationSettingsRestClient.updateNotificationSettings({
+            notify: this.regularNotificationToggle.checked,
+            frequencyInWeeks: this.twoWeeklyFrequencyRb.checked ? 2 : 4,
+            notificationAtReleaseDate: this.releaseDateNotificationToggle.checked,
+            notificationAtAnnouncementDate: this.announcementDateNotificationToggle.checked
+        }).catch(response => {
+            const message = `<h3 class="h5">${UNKNOWN_ERROR_MESSAGE}</h3>Your changes may not have been saved. Please try again later.`;
+            const infoMessage = this.alertService.renderErrorAlert(message, false);
+            this.hostElement.insertAdjacentElement("afterbegin", infoMessage);
+            console.error(response);
```
Comment (hedgehog1833) [inline webapp/src/main/resources/static/ts/src/service/notification-settings-render-service.ts:77]: This is also there deliberately?

Comment (DanielW1987) [reply webapp/src/main/resources/static/ts/src/service/notification-settings-render-service.ts:77]: nope

File: webapp/src/main/resources/static/ts/src/clients/notification-settings-rest-client.ts (hunk 1 above)
Comment (DanielW1987) [reply webapp/src/main/resources/static/ts/src/clients/notification-settings-rest-client.ts:15]: logging only here

**Review by hedgehog1833 — APPROVED** [review]

**Comment by DanielW1987** [comment]
[Make Notifications configurable](https://trello.com/c/hlDkMGvu/255-make-notifications-configurable)

**Pull request merged**